### DISCUSSION TITLE
Missing edge-cleaning function?

### DISCUSSION THREAD

**Discussion by 4irm** [Tasshack/dreame-vacuum#449] [Q&A] [answered]
Dear all, I have just reverted from the dreame app (L10s Ultra) to xiaomi home in order to use my vacuum cleaner in home assistent. 

Unfortunately, the cleaning edge functionality seems to be missing - this makes really a difference in edges and along the wall...

Or do I just don't find the option since xiaomi app does not have this functionality either?

I heard that the beta supports dreame login - I guess this doesn't add this functionality?

![image](https://github.com/Tasshack/dreame-vacuum/assets/28482836/cb99aa88-85e0-459a-a33f-26b90076e377)

Thank you!

**Comment by Tasshack** [comment, marked as answer]
Both wider corner coverage and dreamehome account are available on Beta.
